test/unit/user_test.rb

```diff
@@ -760,21 +760,38 @@ class PasswordStrengthTest < ActiveSupport::TestCase
     setup do
       provider = FactoryBot.create(:simple_provider)
       @buyer = FactoryBot.create(:buyer_account, provider_account: provider)
+      @user_with_password = ->(password) do
+        @buyer.users.new username: 'user', email: 'user@example.com', password: password, password_confirmation: password
```

Comment:
This is to make the resulting user a valid object, except for the password.
So, we don't need just to trigger `user.valid?` we can also assert its value.